pages/[slug].js

```diff
@@ -126,6 +129,23 @@ export default function Post({ postData, status, categoryPosts, mixpanel }) {
     }
   }
 
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+
+    await axios
+      .post(`${config.STRAPI_URL}/v1/user/subscribeUser?populate=deep`, {
+        email,
+      })
+      .then((response) => {
+        setCopied(true);
+        if (response.data.is_subscribed) {
+          setMessage("Subscribe Successfully!");
```

Comment:
Show this message at the top of the page, where we are showing the `Link copied` message. Or you can use sam prompt, by changes messages

Reply:
we are already display `link copied` message at bottom of the page. and i used same alerts ui for messages print,only chnage message.